src/rdkafka_conf.c

```diff
@@ -58,21 +58,22 @@
 struct rd_kafka_property {
         rd_kafka_conf_scope_t scope;
         const char *name;
-        enum { _RK_C_STR,
```

Comment:
avoid whitespace/indent diffs, this is also likely to fail `make style-check`, unless something changed in clang-format.

Reply:
I didn't actually touch these enums. These changes are from running `make style-fix-changed`

Reply:
Oh. that's really strange. Seems to run differently on my box compared to the CI.
I've manually undone it

Reply:
Ugh, that's frustrating (looking at you, clang-format)

Reply:
Looks like it passed this time